frontend/src/components/UseCategoryDropdown.vue

```diff
@@ -1,39 +1,45 @@
 <template>
-  <select @change="$emit('selected', $event.target.value)" class="UseCategoryDropdown">
-    <option value="">Everything</option>
-    <option :value="category" v-for="category in categories">
-      {{ category }} Uses
-    </option>
-  </select>
+<b-dropdown :text="category" class="UseCategoryDropdown" right>
+  <b-dropdown-item-button class="category-item" @click="onCategorySelected('Everything')">Everything</b-dropdown-item-button>
+  <b-dropdown-item-button class="category-item" :value="category" v-for="category in categories" @click="onCategorySelected(category)">
```

Comment:
Add a `:key="category"` to silence these warnings:

```
(Emitted value instead of an instance of Error) <b-dropdown-item-button v-for="category in categories">: component lists rendered with v-for should have explicit keys. See https://vuejs.org/guide/list.html#key for more info.
```